modin/pandas/test/utils.py

```diff
@@ -645,6 +645,7 @@ def eval_general(
     check_exception_type=True,
     raising_exceptions=None,
     check_kwargs_callable=True,
+    modin_warning=False,
```

Comment:
It's better to use `None` here.

Reply:
Let's leave the functionality for checking warnings outside of `eval_general`. It's already the super-mega-power function.

It's a pretty nice as is.
```python
with pytest.warns(UserWarning):
    eval_general(...)
```

Reply:
Ok, both suggestions were implemented.